src/annotator/test/config-test.js

```diff
@@ -1,86 +1,220 @@
 'use strict';
 
-var config = require('../config');
-var unroll = require('../../shared/test/util').unroll;
-
-describe('annotator configuration', function () {
-  var fakeScriptConfig;
-
-  function fakeQuerySelector(selector) {
-    if (selector === 'link[type="application/annotator+html"]') {
-      return {href: 'app.html'};
-    } else if (selector === 'script.js-hypothesis-config' &&
-               fakeScriptConfig) {
-      return {textContent: fakeScriptConfig};
-    } else {
-      return null;
-    }
-  }
-
-  var fakeWindowBase = {
+var proxyquire = require('proxyquire');
+
+var fakeSettings = sinon.stub();
+var fakeExtractAnnotationQuery = {};
+
+var config = proxyquire('../config', {
+  '../shared/settings': fakeSettings,
+  './util/extract-annotation-query': fakeExtractAnnotationQuery,
+});
+var sandbox = sinon.sandbox.create();
+
+function fakeWindow() {
+  return {
     document: {
-      querySelector: fakeQuerySelector,
-      querySelectorAll: function (selector) {
-        var match = fakeQuerySelector(selector);
-        return match ? [match] : [];
-      },
+      querySelector: sinon.stub().returns({href: 'LINK_HREF'}),
     },
-    location: {hash: ''},
+    location: {href: 'LOCATION_HREF'},
   };
+}
 
-  beforeEach(function () {
-    fakeScriptConfig = '';
+describe('src.annotator.config', function() {
```

Comment:
This is slightly inconsistent with the naming convention you used for `links-test.js`. I would suggest we omit the `src` prefix since it would be present for every single test otherwise.